config/routes.rb

```diff
@@ -0,0 +1,57 @@
+Rails.application.routes.draw do
+  # The priority is based upon order of creation: first created -> highest priority.
+  # See how all your routes lay out with "rake routes".
+  #my own root
+  root 'application#goodbye'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.